docs/Project.toml

```diff
@@ -2,4 +2,5 @@
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Do we need Revise as a dependency for our project?

Reply:
Ooh nice catch! No we don't.

Reply:
Fixed it: #900